Firestore/Protos/build_protos.py

```diff
@@ -57,15 +57,21 @@ def main():
       '--nanopb', action='store_true',
       help='Generates nanopb messages.')
   parser.add_argument(
-      '--output-dir', '-d', default='.', dest='output_dir',
+      '--protos-dir', dest='protos_dir',
+      help='Source directory containing .proto files.')
+  parser.add_argument(
+      '--output-dir', '-d', dest='output_dir',
       help='Directory to write files; subdirectories will be created.')
 
   parser.add_argument(
       '--protoc', default='protoc',
       help='Location of the protoc executable')
   parser.add_argument(
-      '--protoc-pythonpath', dest='protoc_pythonpath',
+      '--pythonpath', dest='pythonpath',
```

Comment:
dest is redundant here (since you removed the dash/underscore).

Reply:
Done.